enterprise/models.py

```diff
@@ -118,6 +118,13 @@ class Meta:
         )
     )
 
+    contact_email = models.EmailField(
```

Comment:
It's sort of like the email field in the Django auth_user table -- awful place to store that type of info, imnsho.